package.json

```diff
@@ -2180,6 +2180,11 @@
 					"group": "inline@2",
 					"when": "view =~ /(pr|prStatus):github/ && viewItem =~ /description/"
 				},
+				{
+					"command": "pr.openPullRequestOnGitHub",
+					"group": "inline@3",
+					"when": "view =~ /(pr|prStatus):github/  && viewItem =~ /description/"
```

Comment:
Could we also limit this action to show only when there's more than one active PR? The original issue is that when there's more than one active PR then there's no easy way to open github.com.

Reply:
See here for an example of how to check the count of active PRs:

https://github.com/Santhoshmani1/vscode-pull-request-github/blob/1d12381dd4266025b54ad2942ccea1d4bf890e9b/package.json#L718-L719

Reply:
I have modified the openPrOnGithub Icon visibility to show only when multiple active pull requests are present.